Blish HUD/GameServices/ArcDpsService.cs

```diff
@@ -153,8 +179,10 @@ public class ArcDpsService : GameService {
         }
 
         private void SocketErrorHandler(object sender, SocketError socketError) {
-            var listener = (SocketListener)sender;
-            listener.Stop();
+            // Socketlistener stops by itself.
+            Logger.Error("ArcDpsService has socket error: {0}", socketError.ToString());
```

Comment:
Is `has socket error` worded correctly? I think `encountered` or something along those lines would fit better here

Reply:
Yeah, that sounds wrong. I have changed that to encountered.